pkg/Microsoft.NETCore.Platforms/runtime.json

```diff
@@ -292,18 +292,18 @@
             "#import": [ "ubuntu.14.04", "ubuntu-x64" ]
         },
 
-        "ubuntu.14.10": {
```

Comment:
@ericstj 

What's the plan here?  Do we want this file to have all the platforms we support + the inbetween ones (e.g. 14.10, 15.04, 15.10)?

I also worry a little about the imports here.  I think it would be better if the fallback for specific versions was to "ubuntu" instead of the previous version'd release.